src/App.tsx

```diff
@@ -1,60 +1,153 @@
 import './App.scss';
+import React, { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { FullTodo, Todo } from './types/Todo';
+import { User } from './types/User';
+import { TodoList } from './components/TodoList';
+
+const prepareTodos = (
+  users: User[],
+  todos: Todo[],
+): FullTodo[] => (
+  todos.map((todo) => ({
+    ...todo,
+    user: users.find((user) => user.id === todo.userId) || null,
+  }))
+);
+
+const prepareTodo = (
+  currentUser: User,
+  title: string,
+  todos: Todo[],
+): FullTodo => {
+  const maxId = Math.max(...todos.map(todo => todo.id));
+
+  return ({
+    id: maxId + 1,
+    title,
+    completed: false,
+    userId: currentUser.id,
+    user: currentUser,
+  });
+};
```

Comment:
mb better to create constant in your handle,not using this function,because if you creating this object in handler its makes your code shortets and you not reusing this function